### PR TITLE
Added new parameter and argument "scan_visbility"

### PR DESCRIPTION
**Pull request by aburt-demisto**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/content/issues/17721

## Description
Added a new parameter and argument (to !url) for scan_visibility

## Screenshots
NA

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 1]
```diff
@@ -234,6 +254,17 @@ script:
...
+      defaultValue: 'public'
+      description: The submission visibility, overrides the 'public' argument.
+      predefined:
+        - public
+        - private
+        - unlisted
+      isArray: false
+      name: scan_visibility
+      required: false
+      secret: false
```
Comment (yucohen) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:267]: why are we adding this argument to this command if it's deprecated?

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.py [hunk 2]
```diff
@@ -42,7 +42,7 @@
 
 
 class Client:
-    def __init__(self, api_key='', threshold=None, use_ssl=False, reliability=DBotScoreReliability.C):
+    def __init__(self, api_key='', scan_visibility='public', threshold=None, use_ssl=False, reliability=DBotScoreReliability.C):
         self.base_url = 'https://urlscan.io/api/v1/'
         self.api_key = api_key
         self.threshold = threshold
```
Comment (yucohen) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.py:48]:
```suggestion
        self.threshold = threshold
        self.scan_visibility = scan_visibility
```

**Review by yucohen — CHANGES_REQUESTED** [review]
Looks good! please see my comments.
In addition, please update the `README.md` file and add the new arguments and parameters added

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml (hunk 1 above)
Comment (aburt-demisto) [reply Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:267]: It was a customer request and it also allows a specific submission mode for URLScan that some customers have.

Comment (yucohen) [reply Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:267]: ahh great

Comment (yucohen) [reply Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:267]: @bakatzir what do you think?

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 3]
```diff
@@ -121,6 +130,17 @@ script:
       name: timeout
       required: false
       secret: false
+    - auto: PREDEFINED
+      defaultValue: 'public'
+      description: The submission visibility, overrides the 'public' argument.
```
Comment (bakatzir) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:135]:
```suggestion
      description: The submission visibility. If specified, overrides the 'public' parameter.
```

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 4]
```diff
@@ -234,6 +254,17 @@ script:
       name: url
       required: true
       secret: false
+    - auto: PREDEFINED
+      defaultValue: 'public'
+      description: The submission visibility, overrides the 'public' argument.
```
Comment (bakatzir) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:259]:
```suggestion
      description: The submission visibility. If specified, overrides the 'public' parameter.
```

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml (hunk 1 above)
Comment (bakatzir) [reply Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:267]: LGTM

File: Packs/UrlScan/ReleaseNotes/1_1_16.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### urlscan.io
+- Added the "scan _visbility" to the url command
```
Comment (bakatzir) [inline Packs/UrlScan/ReleaseNotes/1_1_16.md:4]:
```suggestion
- Added the "*scan _visibility* argument to the ***url*** command.
```

File: Packs/UrlScan/pack_metadata.json [hunk 6]
```diff
@@ -2,7 +2,7 @@
     "name": "URLScan.io",
     "description": "urlscan.io Web Threat Intelligence",
     "support": "partner",
-    "currentVersion": "1.1.15",
+    "currentVersion": "1.1.16",
```
Comment (bakatzir) [inline Packs/UrlScan/pack_metadata.json:5]: should be a minor version bump https://semver.org/

**Review by bakatzir — COMMENTED** [review]
please addd to the readme as well.
as it is OOTB XSOAR supported content, I leave the BC checks/UT to Yuval.

**Review by yucohen — APPROVED** [review]
will handle readme in the internal

**Pull request merged**